app/internal/weekly_parasha.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+from typing import Dict, Optional
+
+from app.database.models import Parasha
+from sqlalchemy.orm import Session
+
+
+def create_parasha_object(parashot_fields: Dict[str, Optional[str]])\
+        -> Parasha:
```

Comment:
Use git precommit hooks to fix the formatting here

Reply:
i change it